docs/Reference/Config-Items.md

```diff
@@ -83,16 +83,17 @@ In private networks, the milestone block defines the protocol version for the ne
     {
       "config": {
         ...
-        "constantinopleFixBlock": 0,
+        "londonBlock": 0,
         ...
       },
     }
     ```
 
 !!! tip
 
-    When specifying the milestone block for private networks, you only need to specify the latest
+    When specifying the milestone block for private networks, we recommend specifying the latest
```

Comment:
We recommend specifying the latest milestone block for private networks.
or
When starting a new private network, we recommend specifying the latest milestone

Reply:
addressed